tests/unit_tests/emission_curve.cpp

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2014-2018, The Monero Project
```

Comment:
Wrong cc and licence (I assume you wrote 100% of this, correct?)

Reply:
updated, since I copied the old emission curve into the test I keeped the original header and added our